### DISCUSSION TITLE
Plugin not showing wallet balance.

### DISCUSSION THREAD

**Discussion by karega** [rainbow-me/rainbowkit#2087] [Feedback]
The plugin isn't showing the current wallet balance and I am receiving various errors and warnings.
![image](https://github.com/rainbow-me/rainbowkit/assets/190614/65538011-cf0c-493a-a4b4-a4ee5aed0aca)
![image](https://github.com/rainbow-me/rainbowkit/assets/190614/64724a1a-9e19-4dac-a168-52d3b38166c8)

**Comment by DanielSinclair** [comment]
These errors actually appear to be from Coinbase Wallet. It sounds like your issue could be your dApp's public provider. Make sure that you're using a valid transport and API key. Public providers will often be unreliable, so it is typically better to use Alchemy or similar private node. https://wagmi.sh/react/api/transports/http
